Fix: class method naming in document data extractor

albertmueller opened the pull request:
## Changes in this pull request
Change method name from addDoumentEditables to addDocumentEditables

albertmueller committed:
fix(core): fix class method naming in document data extractor

kingjia90 commented on bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php:
```diff
@@ -61,7 +61,7 @@ public function extract(TranslationItem $translationItem, string $sourceLanguage
      *
      * @throws \Exception
      */
-    protected function addDoumentEditables(Document $document, AttributeSet $result): DocumentDataExtractor
```
can you please also add BC layer, so that `addDoumentEditables` returns `addDocumentEditables`. 
The former wrong method needs also a phpdoc deprecation since 11.2 (we are releasing 11.1 right now, not sure would make on time) and removal in Pimcore 12.

albertmueller replied:
done :)

albertmueller committed:
chore(core): adding bc layer

BREAKING CHANGE: Using addDoumentEditables is deprecated and will be removed in Pimcore 12, use addDocumentEditables instead.

albertmueller committed:
chore(sonar): reduce amount of characters in a line

kingjia90 commented on bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php:
```diff
@@ -50,18 +50,29 @@ public function extract(TranslationItem $translationItem, string $sourceLanguage
+    {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '11.2',
```
oh....i had a mix up, actually we are releasing `admin-ui-classic-bundle` `1.1`, not pimcore `11.1`, so my previous comment was wrong and should be
```suggestion
            '11.1',
```

albertmueller committed:
Update bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php

approve suggestion

Co-authored-by: JiaJia Ji <kingjia90@gmail.com>

kingjia90 committed:
Merge remote-tracking branch 'origin/11.x' into fix/fix-class-method-naming-in-document-data-extractor
